tests/console/login.pm

```diff
@@ -19,5 +19,9 @@ sub run {
     my $self = shift;
     select_console 'root-ssh';
     record_info("console logined");
+    my $guests_str = script_output("echo \$(grep test_guest /etc/hosts|cut -d ' ' -f2)");
```

Comment:
I am afraid you use "/etc/hosts" file in an improper way. You can not use it as a store your parameters I think. /etc/hosts is a mapping from dns to ip address. I think you should use the other way to achieve your purpose.